src/public/locales/fr/default_translation_fr.json

```diff
@@ -77,5 +76,15 @@
   "or": "ou",
   "Ingestion History Is Empty": "L'historique d'ingestion est vide",
   "Latest Data Ingestion": "Dernière ingestion de données",
+  "Beacon error": "Erreur de Beacon",
+  "variants form should include either an end position or both reference and alternate bases": "le formulaire de recherche de variants doit inclure soit la position de fin, ou la base de référence et la base alternative.",
```

Comment:
I think it makes more sense to capitalize this in both EN and FR and then fix the lookup string in BeaconQueryUi